gen/definitions/classes.yaml

```diff
@@ -20,6 +23,7 @@ vzCPIf:
 rtctrlProfile:
   resource_name: "route_control_profile"
   rn_format: "/prof-{name}"
+  exclude: true
```

Comment:
Why is exclude used here? Did we introduce the ability to not generate resources if exclude is set to true in the generator and its meta is included?